components/dashboard/src/data/projects/set-project-env-var-mutation.ts

```diff
@@ -5,16 +5,22 @@
  */
 
 import { useMutation } from "@tanstack/react-query";
-import { getGitpodService } from "../../service/service";
+import { envVarClient } from "../../service/public-api";
+import { EnvironmentVariableAdmission } from "@gitpod/public-api/lib/gitpod/v1/envvar_pb";
 
 type SetProjectEnvVarArgs = {
     projectId: string;
     name: string;
     value: string;
-    censored: boolean;
+    admission: EnvironmentVariableAdmission;
```

Comment:
please update setup file to bump cache version and list env var classes to be safe

Reply:
code refer
- cache version [[1](https://github.com/gitpod-io/gitpod/blob/599e7b95c3522e53da2fa5bf8d65d643cfb02f61/components/dashboard/src/data/setup.tsx#L29)]
- classes [[2](https://github.com/gitpod-io/gitpod/blob/599e7b95c3522e53da2fa5bf8d65d643cfb02f61/components/dashboard/src/data/setup.tsx#L147)]